egs/icsi/README.txt

```diff
@@ -0,0 +1,99 @@
+
+To train models, for an arbitrary mic, go to s5 run something like (after you
```

Comment:
can you please move the "instructional" type recipes to a comment at the top of the run.sh, and have this README.txt more about the data itself?  Please prominently say how many hours of train/test, what language, what type of speech (I assume meeting, with overlaps and multiple microphones), and what sampling rate.

Reply:
... and also make clear prominently, at the top, that it's available from the LDC and mention the corpus numbers.
The idea is that people who are looking for setups with free data will know quickly not to look hdre.